src/appellate/appellate.js

```diff
@@ -35,8 +35,33 @@ AppellateDelegate.prototype.dispatchPageHandler = function () {
 
 AppellateDelegate.prototype.handleCaseSelectionPage = async function () {
   // retrieve pacer_case_id from the link related to the Case Query
-  this.pacer_case_id = APPELLATE.getCaseIdFromCaseSelection();
-  await saveCaseIdinTabStorage({ tabId: this.tabId }, this.pacer_case_id);
+  if (document.getElementsByName('csnum1')[0].value || document.getElementsByName('csnum2')[0].value){
```

Comment:
I don't think Appellate PACER has a page similar to the **"Select a Person" page**. I tried to search cases using 
 the **Party/Attorney field**, the results were always shown using the **Case Selection page**.

We need this `if` statement to avoid searching the **pacer_case_id** in the **Case Selection page**( in Appellate) when there's more than one row on the page(like the last screenshot from the PR summary). We want to avoid searching the **case_id** because those pages contain results from multiple cases.

After inspecting the `body` of this page, I've found that these hidden inputs (the `csnum1` and `csnum2`) were populated in the following cases;
 
 - When the **Case Selection Page** was listing one case ( `csnum1` is populated, `csnum2` is not populated).
 - When the **Case Selection Page** was listing two cases (both hidden inputs were populated).

The lines of code inside the `if` block should be executed when **only one case** is listed on the page so we can save its pacer_case_id in the tab storage and use it in the docket reports.

Thanks to your comment, I realized that the condition of the `if` statement is wrong and should be:

``` javascript 
if (document.getElementsByName('csnum1')[0].value && !document.getElementsByName('csnum2')[0].value){

// The statements inside this block should be executed only when the page lists one case.

}
```

Reply:
Cool. In that case, I think it's worth it to wrap these checks in a function with a useful name and comment, so that it's obvious in a month what's going on here. I live in fear of node selectors that I can't explain in the future.